app/controllers/session_controller.rb

```diff
@@ -0,0 +1,12 @@
+class SessionController < ApplicationController
+  def create
+    user = User.find_by(username: params[:session][:username])
+    if user && user.authenticate(params[:session][:password])
+      session[:user_id] = user.id
+      redirect_to root_url
+    else
+      flash[:errors] = 'Invalid Login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.